composer.json

```diff
@@ -19,19 +19,19 @@
     "composer/semver": "^1.4",
     "graylog2/gelf-php": "^1.4.2",
     "guzzlehttp/guzzle": "^6.2",
-    "illuminate/config": "^5.6",
+    "illuminate/config": "^5.5",
     "magento/magento-cloud-components": "^1.0.1",
     "magento/magento-cloud-docker": "^1.0.0",
     "magento/magento-cloud-patches": "^1.0.0",
     "monolog/monolog": "^1.16",
     "nesbot/carbon": "^1.0||^2.0",
     "psr/container": "^1.0",
     "psr/log": "^1.0",
-    "symfony/config": "^3.4||^4.3",
-    "symfony/console": "^2.3||^4.0",
-    "symfony/dependency-injection": "^3.4||^4.3",
+    "symfony/config": "^4.4",
+    "symfony/console": "^2.8||^4.0",
+    "symfony/dependency-injection": "^2.8||^4.3",
```

Comment:
Here we need at least `symfony/dependency-injection: ^3.4||^4.3`.
With `^2.8` ece-tools fails with error:
```
       Fatal error: Uncaught Symfony\Component\Config\Util\Exception\XmlParsingException: ERROR 1871 Element '{http://symfony.com/schema/dic/services}defaults': This element is not expected. Expected is ( {http://symfony.com/schema/dic/services}service ). (in /app/ - line 7, column 0)
```